content/downloads/_index.md

```diff
@@ -35,15 +35,46 @@ For full instructions and prerequisites, see [GoFSH Installation](/docs/gofsh/in
 
 ### Presentations
 
-* <a href="https://youtu.be/RfmqpUA606U">Introduction to FHIR Shorthand</a>, from FHIR DevDays, November 2020
-  * Slides <a href="201118_Mark_Kramer_Introduction_To_FHIR_Shorthand.pdf">here</a>
-* <a href="https://youtu.be/7yzLzQjict0">Let's Build - Profiling with FHIR Shorthand</a>, from FHIR DevDays, November 2020
-  * Slides <a href="201118_Chris_Moesel_Lets_Build_with_FHIR_Shorthand.pdf">here</a>
-* <a href="https://youtu.be/Ohgx5TedLlI">Advanced FHIR Shorthand and Tools</a>, from FHIR DevDays, June 2021
-  * Slides <a href="20210610_Chris_Moesel_Advanced_FSH_And_Tools.pdf">here</a>
+* <a href="https://youtu.be/tmtEnjxrCDA">Learn to FSH: A friendly introduction to FHIR Shorthand</a>, from FHIR DevDays, June 2023
+  * Slides <a href="230606_MarkKramer_Learn_to_FSH.pdf">here</a>
```

Comment:
Is there a reason you have uploaded these presentations to FSH School instead of directly linking to them on the DevDays site? For example, this presentation is available here: https://www.devdays.com/wp-content/uploads/2023/08/230606_MarkKramer_Learn_to_FSH.pdf

Reply:
I uploaded the presentations just to align with the other presentations that were already there. I'm not opposed to using the direct links for all of them though.

Reply:
I updated all the DevDays presentations to link to the presentation on the DevDays site.

However, I couldn't find the Let's Build slides from June 2021 on the DevDays site. There were two sets of the slides from the Create an IG with FSH presentation from that year, so I have a feeling that presentation got uploaded twice and the Let's Build didn't get uploaded. Since we had the slides uploaded to FSH School already, I kept those for that one presentation.